dwave/system/composites/reversecomposite.py

```diff
@@ -142,6 +186,29 @@ class ReverseBatchStatesComposite(dimod.ComposedSampler):
        sampler (:obj:`dimod.Sampler`):
             A dimod sampler.
 
+    Examples:
+       This example run 100 reverse anneals each from two initial states on a problem
```

Comment:
Found and replace four, thanks!

Reply:
replaced? :smile:

Reply:
![image](https://user-images.githubusercontent.com/34041130/93246253-62c16d80-f741-11ea-92a3-efeae3e89a10.png)
I have absolutely no idea what you are talking about.

Reply:
Maybe this will jog your memory: :stuck_out_tongue: 

![image](https://user-images.githubusercontent.com/447371/93246646-f09d5880-f741-11ea-9ca6-97e1bfc1b032.png)